pkg/service/node.go

```diff
@@ -114,31 +126,34 @@ func (n *NodeService) NodeStageVolume(_ context.Context, req *csi.NodeStageVolum
 	}
 	klog.V(3).Infof("Staging volume %s", req.VolumeId)
 
-	if req.VolumeCapability.GetMount() != nil {
-		// Filesystem volume mode, create FS if needed
-		// get the VMI volumes which are under VMI.spec.volumes
-		// serialID = kubevirt's DataVolume.UID
-		device, err := getDeviceBySerialID(req.VolumeContext[serialParameter], n.deviceLister)
-		if err != nil {
-			klog.Errorf("Failed to fetch device by serialID %s", req.VolumeId)
-			return nil, err
-		}
+	if req.VolumeCapability.GetMount() == nil {
```

Comment:
sorry for this mess, forgot I change this. It's just flipping the logic to return early for better indentation
using `if req.VolumeCapability.GetMount() == nil {`. no logical change